src/test/regress/expected/upgrade_citus_finish_citus_upgrade_0.out

```diff
@@ -0,0 +1,30 @@
+-- Citus upgrades are finished by calling a procedure
```

Comment:
Why was this extra expected output needed?

Reply:
Because when upgrading from 11.3, finish_citus_upgrade behaves differently.
However I don't want to remove the upgrading from 11.3 since we have dedicated upgrade test for it.

Reply:
It is a good practice to have an output of the PG version in the beginning of a test file in case we need alternative outputs for different PG versions.

This would have made it easier for the future reader to understand why we have an alternative test output.

Reply:
Good point. Let me add that before I merge.

Reply:
How do we ensure that we always `run upgrade to 11` test? Now, there is no such guarantee, the original output file is NOT guaranteed to be used.

That is a pretty critical test, and if possible, I'm fine to skip running the test for upgrades that are no-op and remove this additional output file.

Do you see a way for that?

Reply:
I do not really follow what you mean @onderkalaci. This test is listed in the very beginning of `after_citus_upgrade_coord_schedule` meaning that it is run before any of the other tests after an upgrade.

Both of the alternative files actually run the same set of commands on the cluster. In case you haven't noticed, the newly introduced version checks in this file are not there to skip the remainder of the tests if a conditional on the version is satisfied. This is not the case for `upgrade_post_11_after` where we early exit in case it does not really make sense to test if the the version before the upgrade was not less than 11.0.

Reply:
@onderkalaci 
> How do we ensure that we always run upgrade to 11 test? 

Other than what @hanefi said, I'd like to add that we _are sure_ we run upgrades starting from 10.2 to current, which includes upgrade to 11 path.
And we are also sure we run upgrades starting from 11.3 to current, which doesn't include upgrade to 11 path. Given that we run the same upgrade tests, there has to be an alternative file output.

https://github.com/citusdata/the-process/blob/master/circleci/images/Makefile
`CITUS_UPGRADE_VERSIONS=v10.2.0 v11.3.0`